README.md

```diff
@@ -10,11 +10,11 @@ This project is focused on the integration layer that is required to enable exec
  Host modules can call wasm functions directly. Pass function name of wasm chaincode as argument in a transaction
 
 How WASM Chaincodes are stored in hyperledger fabric by wasmcc:
-![Wasm storage](https://github.com/kleash/fabric-chaincode-wasm/blob/master/docs/wasm-chaincode-create.png)
+![Wasm storage](https://github.com/hyperledger-labs/fabric-chaincode-wasm/blob/master/docs/wasm-chaincode-create.png)
```

Comment:
Instead of having all these links, you can also just link them relatively since the file is in the same repo. I think that may be better. That applies to most of the links here I believe.

Reply:
better idea!